pkgs/kde/gear/calindori/default.nix

```diff
@@ -0,0 +1,15 @@
+{
+  mkKdeDerivation,
+  qtsvg,
+  qqc2-desktop-style,
+}:
+mkKdeDerivation {
+  pname = "calindori";
+
+  extraBuildInputs = [
+    qtsvg
+    qqc2-desktop-style
+  ];
+  # FIXME(qt5)
```

Comment:
I am confused, isn't plasma6 qt6?

Reply:
Plasma was fully ported, but Gear wasn't. Some applications are still on Qt5 and will catch up in future releases, and so is Plasma Bigscreen.